### DISCUSSION TITLE
Pagination links return error 500

### DISCUSSION THREAD

**Discussion by JJaraser** [livewire/livewire#5578] [1. Help]
Hello Everyone.

I´m using laravel v7.30.6 and livewire v1.3.9

I have a website that has a list of products and about 2 months ago the pagination started to fail.

The first page renders correctly but when people try to go to the next page it generates a 500 error.

I enabled the debug to check the error and this is the error i get.
![error_livewire](https://user-images.githubusercontent.com/120282099/221186767-f4729e09-105c-4727-b11b-d103309cbaca.PNG)

does anybody knows what that means.

Any help will be appreciated

**Comment by dennisprudlo** [comment]
This seems to be a problem with `facade/ignition` since this is the package throwing the exception not livewire. Did you recently (or 2 months ago) updated your dependencies including ignition?

The `LivewireManager::originalMethod` method doesn't exist in Livewire 1.3.9, thats why the current version of `facade/ignition` requires Livewire ^2.4/^2.5. So upgrading Livewire or downgrading `facade/ignition` should do the trick.

Either way I'm surprised you are even able install Livewire 1.3.9 and one of those `facade/ignition` versions. Are you sure you have Livewire 1.3.9 installed? What exactly are your version constraints in your composer.json?

**Reply by joshhanley** [reply]
@JJaraser yeah Dennis is correct here, I would look at upgrading or downgrading as suggested.

It looks like ignition doesn't have any version checks for which version of Livewire is installed, so that's probably why it was installed without any issues. So you will need to manage that yourself and lock ignition to an older version if you can't upgrade Livewire.
